MapleServer2/PacketHandlers/Game/UserChatHandler.cs

```diff
@@ -5,21 +5,25 @@
 using MapleServer2.Tools;
 using Microsoft.Extensions.Logging;
 
-namespace MapleServer2.PacketHandlers.Game {
-    public class UserChatHandler : GamePacketHandler {
+namespace MapleServer2.PacketHandlers.Game
+{
+    public class UserChatHandler : GamePacketHandler
+    {
         public override ushort OpCode => RecvOp.USER_CHAT;
 
         public UserChatHandler(ILogger<GamePacketHandler> logger) : base(logger) { }
 
-        public override void Handle(GameSession session, PacketReader packet) {
-            var type = (ChatType) packet.ReadInt();
+        public override void Handle(GameSession session, PacketReader packet)
+        {
+            ChatType type = (ChatType)packet.ReadInt();
             string message = packet.ReadUnicodeString();
-            string recipient = packet.ReadUnicodeString();
+            packet.ReadUnicodeString(); // Recipient
```

Comment:
Again why remove the variable

Reply:
The variable is not being used, so for not lose track, i added as comment to remember what it was